src/jdk.javadoc/share/classes/jdk/javadoc/doclet/Reporter.java

```diff
@@ -25,41 +25,125 @@
 
 package jdk.javadoc.doclet;
 
+import java.io.PrintWriter;
+import java.util.Locale;
 import javax.lang.model.element.Element;
 import javax.tools.Diagnostic;
+import javax.tools.FileObject;
 
 import com.sun.source.util.DocTreePath;
 
 /**
- * This interface provides error, warning and notice reporting.
+ * Interface for reporting diagnostics and other messages.
+ *
+ * <p>Diagnostics consist of a {@link Diagnostic.Kind diagnostic kind} and a message,
+ * and may additionally be associated with an {@link Element element},
+ * a {@link DocTreePath tree node} in a documentation comment,
+ * or at an arbitrary position in a given {@link FileObject file}.
```

Comment:
"at" looks misplaced here.

Reply:
When I wrote it, it looked right;  now, I agree with you.